cmd/maya-apiserver/spc-actions/cstorpool_delete.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2017 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cstorpool
+
+import (
+	"github.com/golang/glog"
+	"fmt"
+	"github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	"github.com/openebs/maya/pkg/storagepool"
+)
+
+
+func DeleteCstorpool(key string) (error) {
```

Comment:
CreateCStorPool receives a SPC object, why is DeleteCStorPool receiving only a string?

Reply:
For creation of a object we need more information other than name.
For deletion, name of the resource is the only information that is used to perform deletion.
Hence it is so.